bin/rouge_cmd.py

```diff
@@ -50,7 +50,7 @@ def main():
         assert(type(ref) == str)
 
         rouge = Rouge()
-        scores = rouge.get_scores(hyp, ref, metrics, stats, avg=args.avg)
+        scores = rouge.get_scores(hyp, ref, avg=args.avg)
```

Comment:
Does this raise an error? 
It should be fixed, not just removed, it serve a purpose

Reply:
The called function doesn't have those parameters so the current code in the repo doesn't even compile. Perhaps you forgot to commit the new/updated version of the function when you updated this function call.

Reply:
Oh yeah, my bad. `metrics` and `stats` are parameters of the `Rouge` object, just one line above.

Reply:
Seems to be an easy fix then. I'll have a look at it in the evening.

Reply:
I've just noticed that unfortunately I no longer have the fork of this repo. I deleted it because I needed something quickly and this repo seemed to be inactive for days after my PR.

Reply:
OK no problem I'll fix that in another PR. 
The repo isn't inactive in itself, I just had holidays past weeks :)